### ISSUE TITLE
Files Must Be Located In 'src' Folder

### ISSUE THREAD

**Issue by grumpytechdude** [kevinohara80/dmc#37]
Our SF Code is located in several folders, one of which is called 'source', another of which is called 'systemtests'. It'd be great if dmc could support specifying source folders!
Currently, if you try and deploy a file that does not exist in the 'src' folder (which for us doesn't exist), you get
`$ dmc deploy source/classes/StagingVersionDomain.cls
[dmc] using org: [orgname] (default)
[dmc] searching for local metadata
[dmc] deploying 1 metadata files
[err] unrecognized file path: source/classes/StagingVersionDomain.cls
[dmc] [NOT OK]`

Symlinks don't seem to work, the file has to explicitly be in the 'src' folder:
`$ dmc deploy src/classes/StagingVersionsTest.cls
[dmc] using org: [orgname] (default)
[dmc] searching for local metadata
[dmc] deploying 1 metadata files
[dmc] deploy mode: dynamic
[dmc] deploy api: tooling
[dmc] loading related metadata ids
[dmc] loaded 1 ids
[dmc] creating stub files
[dmc] created 0 stub files
[dmc] creating static resources
[dmc] deployed 0 static resources
[dmc] creating container
[dmc] [create] metadata container: 1dc58000000IvQCAA0
[dmc] [create] container member: ApexClass::StagingVersionsTest
[dmc] Deploying...
[dmc] * deploy status: Queued`

**Comment by kevinohara80** [comment]
Yep, I thought about adding that as a config option. I just haven't gotten to that one yet. I'm open to pull requests if you want to take a crack at it.
